### DISCUSSION TITLE
No module named 'pyautogui' after using pyinstaller to generate executable file

### DISCUSSION THREAD

**Discussion by jou-hui** [pyinstaller/pyinstaller#5748] [Help] [answered]
I am facing issues after I have used pyinstaller to generate executable output. The error showed no module named 'pyautogui' but I able to run my python code in Jupyter Notebook. I have searched through https://github.com/pyinstaller/pyinstaller/issues/5178, stackoverflow, etc. but I still cannot solve this issues.

Thank you.
Regards,
Jo

**Comment by rokm** [comment, marked as answer]
> but I able to run my python code in Jupyter Notebook

Can you run it with python interpreter (without Jupyter) using the same environment as the one from which you're running PyInstaller?

**Reply by jou-hui** [reply]
I try to run it in Pycharm using the same virtual environment, and it shows that I can run it in Pycharm but cannot run after Pyinstaller

**Reply by rokm** [reply]
Is PyInstaller installed in the same virtual environment, though? I seem to recall a doc enhancement issue that seems to suggest that this is not the case (#5632). So double check if `pyautogui` is installed within the same virtual environment as PyInstaller.

**Reply by jou-hui** [reply]
Yes. I manage to solve this error by installing pyautogui within the same virtual environment as PyInstaller.
